### PR TITLE
Expose launch template override for instance types

### PR DESCRIPTION
**Pull request by fiunchinho**
### What this PR does / why we need it
Towards https://github.com/giantswarm/roadmap/issues/2984

### Checklist

- [X] Update changelog in CHANGELOG.md.

### Trigger e2e tests

/run cluster-test-suites

### CODE REVIEW

File: helm/cluster-aws/values.schema.json [hunk 1]
```diff
@@ -80,6 +80,16 @@
                     "type": "string",
                     "title": "EC2 instance type"
                 },
+                "instanceTypeOverrides": {
```
Comment (fiunchinho) [inline helm/cluster-aws/values.schema.json:83]: I used this name `instanceTypeOverrides`. Open to suggestions if you think it can be improved.

**Comment by fiunchinho** [comment]
Working on the docs here https://github.com/giantswarm/docs/pull/1991

File: helm/cluster-aws/values.schema.json (hunk 1 above)
Comment (calvix) [reply helm/cluster-aws/values.schema.json:83]: The description could be more verbose, does it mean that he original instance is still used, or it isn't? so should I include it again in the override?

Comment (fiunchinho) [reply helm/cluster-aws/values.schema.json:83]: I added more info, and a link to our docs where [this page will be published](https://github.com/giantswarm/docs/pull/1991).

**Review by calvix — APPROVED** [review]

**Pull request merged**